### PR TITLE
Cleaning up upon test completion

### PR DESCRIPTION
**Pull request by bhariharan**
Some of these tests weren't cleaning up after.

### CODE REVIEW

File: libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java [hunk 1]
```diff
@@ -390,6 +390,9 @@ public Bitmap getProfilePhoto() {
      * Fetches this user's profile photo from the server and stores it in the cache.
      */
     public void downloadProfilePhoto() {
```
Comment (wmathurin) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java]: That code came in 6.2, could it throw exception for apps on 6.2?

Comment (bhariharan) [reply libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java]: Nope. `photoUrl` can never really be `null`. Our dummy test account doesn't have that field populated locally. I could add that field in the dummy account.

Comment (wmathurin) [reply libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java]: Agreed, the dummy test account should be more realistic.

Comment (bhariharan) [reply libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java]: Will do.

**Pull request merged**